enterprise/j2ee.core/test/unit/src/org/netbeans/api/j2ee/core/ProfileTest.java

```diff
@@ -59,11 +59,12 @@ public void testFromPropertiesString() {
         assertEquals(Profile.JAKARTA_EE_9_1_FULL, Profile.fromPropertiesString("JAKARTA_EE_9_1_FULL"));
         assertEquals(Profile.JAKARTA_EE_9_1_WEB, Profile.fromPropertiesString("9.1-web"));
         assertEquals(Profile.JAKARTA_EE_9_1_WEB, Profile.fromPropertiesString("JAKARTA_EE_9_1_WEB"));
-        assertEquals(Profile.JAKARTA_EE_10_FULL, Profile.fromPropertiesString("10.0"));
+        assertEquals(Profile.JAKARTA_EE_10_FULL, Profile.fromPropertiesString("10"));
```

Comment:
todo: we have to check what versioning scheme we want

either the tests were wrong or the impl